tests/func/experiments/test_show.py

```diff
@@ -598,6 +598,7 @@ def test_show_only_changed(tmp_dir, dvc, scm, capsys):
     assert "bar" not in cap.out
 
 
+@pytest.mark.pcp
```

Comment:
For this feature, I have been using this custom marker locally. Does it make sense for you to include this kind of custom markers in the repo?

Reply:
@daavoo Could you clarify the purpose?

Reply:
For some features like the pcp (parallel coordinates plot), the related tests are spread across different files and I use the custom marker to have a quick way of running all the tests related to the feature.

Reply:
So during development, I run `pytest -m pcp`

Reply:
The idea to include in repo is that if someone touches this feature in the future, related tests can be run without knowing where all the related files are located.

Reply:
as an alternative if the tests are named semi-consistently then you can do `pytest -k "pcp or parallel_coordinates"` and markers aren't needed.

Reply:
`-k` is a good alternative, for sure. 

IMO, the issue with `-k` is that a feature might be affected by some function with unit tests that don't make sense to name after the feature.

`-m` is explicit, people can see it but `-k` patterns (unless documented) might be tricky if not everybody is aware of the usage.

Reply:
But we're not talking about `-m` versus `-k` in general, right? Just in the case of PCP tests it seems `pytest -k "pcp or parallel_coordinate"` is future-proof & intuitive for devs working on it?